### ISSUE TITLE
why is there a `Users` folder in the Public one?

### ISSUE THREAD

**Issue by eugelogic** [eugelogic/matilda#5]

**Comment by crgeary** [comment]
That is odd. I suspect that is the Users folder of your mac. But this doesn't happen for me at all.

Can you share a screenshot of the CLI output when you run `npm run build` ?

**Comment by eugelogic** [comment]
```
Empty: /public
Copy:  /static => /public
Read:  /content/.DS_Store
Read:  /content/about.html
Read:  /content/image.html
Read:  /content/index.html
Read:  /content/category/benches.html
Read:  /content/gallery/benches.html
Write: /public/about/index.html
Write: /public/image/index.html
Write: /public/index/index.html
Write: /public/category/benches/index.html
Write: /public//Users/Eugene/Documents/JavaScript/0-projects/matilda-master/content/.DS_Store/index.html
Write: /public/gallery/benches/index.html
```

**Comment by crgeary** [comment]
Fixed with #6 .. was the same issue.
